Warn about unwanted mass update. (#883)
cap-js/docs: java/working-with-cql/query-api.md

Use the [Update](../../cds/cqn#update) statement to update existing entities with new data. The update data can be partial (patch semantics), elements without update values keep their old value, except for elements annotated with `@cds.on.update`, which are updated with the annotation value.

> Depending on the filter condition, the `Update` can target [](#update--entities) or [multiple](#searched-update) entity records.

Reviewer: ```suggestion
Depending on the filter condition, the `Update` can target [individual](#update-individual-entities) or [multiple](#searched-update) entity records.
```

   .data("title", "CAP Matters");
```

> ::: warning

Reviewer: ```suggestion
::: danger
```

> If key values are not contained in the data and no filter (`where`, `byId`, `matching`) is specified a [searched update]([url](#searched-update)) will be performed, which updates _all_ entities with the given data.

Reviewer: ```suggestion
If key values are not contained in the data and no filter (`where`, `byId`, `matching`) is specified a [searched update](#searched-update) is performed, which updates _all_ entities with the given data.
```

> If a key value is neither contained in the data nor specified by a filter (`where`, `byId`, `matching`) a [searched update]([url](#searched-update)) will be performed, which updates _all_ entities with the given data.

Author: ```suggestion
If a key value are not contained in the data and no filter (`where`, `byId`, `matching`) is specified a [searched update]([url](#searched-update)) will be performed, which updates _all_ entities with the given data.
```

Reviewer: or just
```suggestion
If the key values are not contained in the data and no filter (`where`, `byId`, `matching`) is specified an unconditional update is performed, which updates _all_ DB entries of the entity with the given data.
```

> If a key value is neither contained in the data nor specified by a filter (`where`, `byId`, `matching`) a [searched update]([url](#searched-update)) will be performed, which updates _all_ entities with the given data.

Reviewer: ```suggestion
If key values are neither contained in the data nor specified by a filter (`where`, `byId`, `matching`) a [searched update](#searched-update) is performed, which updates _all_ entities matching the filter condition with the given data.
```

Author: or

```suggestion
If key values are not contained in the data and no filter (`where`, `byId`, `matching`) is specified a [searched update]([url](#searched-update)) will be performed, which updates _all_ entities with the given data.
```
